src/service.ts

```diff
@@ -147,7 +147,7 @@ export class DebugService implements IDebugger, IDisposable {
    * Whether there exists a thread in stopped state.
    */
   hasStoppedThreads(): boolean {
-    return this._model?.stoppedThreads.size !== 0 ?? false;
+    return this._model?.stoppedThreads.size > 0 ?? false;
```

Comment:
This was an issue because when `this._model` is `null`, we would perform `null !== 0`, which is `true`.